Android/SquareOtto/source/Square.Otto/Square.Otto.csproj

```diff
@@ -0,0 +1,39 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<Project Sdk="MSBuild.Sdk.Extras/3.0.38">
+	<PropertyGroup>
+		<TargetFrameworks>MonoAndroid8.0;MonoAndroid9.0;MonoAndroid10.0;MonoAndroid11.0</TargetFrameworks>
+		<IsBindingProject>true</IsBindingProject>
+		<AssemblyName>Square.Otto</AssemblyName>
+		<RootNamespace>Square.Otto</RootNamespace>
+		<AndroidUseLatestPlatformSdk>False</AndroidUseLatestPlatformSdk>
+		<AndroidUseIntermediateDesignerFile>True</AndroidUseIntermediateDesignerFile>
+		<AndroidClassParser>class-parse</AndroidClassParser>
+		<AndroidCodegenTarget>XAJavaInterop1</AndroidCodegenTarget>
+	</PropertyGroup>
+	<PropertyGroup>
+		<PackageId>Square.Otto</PackageId>
+		<Title>Otto bindings for Xamarin.Android</Title>
+		<Summary>Otto - An event bus by Square</Summary>
+		<Description>An enhanced Guava-based event bus with emphasis on Android support.</Description>
+		<Authors>Microsoft</Authors>
+		<Owners>Microsoft</Owners>
+		<Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+		<PackageLicenseFile>License.md</PackageLicenseFile>
+		<PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
+		<PackageTags>Otto square square.otto xamarin android monodroid</PackageTags>
+		<PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=2130312</PackageProjectUrl>
```

Comment:
```suggestion
		<PackageProjectUrl>https://go.microsoft.com/fwlink/?linkid=2174733</PackageProjectUrl>
```

Reply:
I created a link that takes people to the repo instead of a random folder that we never fixed...